src/extensions/yfm/ImgSize/ImagePaste/index.ts

```diff
@@ -39,6 +41,28 @@ export const ImagePaste: ExtensionAuto<ImagePasteOptions> = (builder, opts) => {
                                     opts,
                                 ).run();
                                 return true;
+                            } else {
+                                const imageUrl =
+                                    e.clipboardData?.getData(DataTransferType.Text) ?? '';
+
+                                if (!imageUrlRegex.test(imageUrl)) {
+                                    return false;
+                                }
+
+                                e.preventDefault();
+
+                                const imageNode = imageType(view.state.schema).create({
+                                    src: imageUrl,
+                                    alt: imageUrl.match(imageNameRegex)?.[1],
+                                });
+
+                                view.dispatch(
+                                    view.state.tr.insert(view.state.tr.selection.from, [
```

Comment:
Why is a `view.state.tr.insert` and a `view.state.schema.text(' ')` used here, ideally the code should be reduced to one view, as [here](https://github.com/gravity-ui/markdown-editor/pull/464/files#diff-550af0eac60195e2f489e0ca052ef38313c36973ddcf71a2adde42412e9ee4bdR24)